src/borg/archiver.py

```diff
@@ -4311,6 +4311,8 @@ def define_borg_mount(parser):
                                help='Set storage quota of the new repository (e.g. 5G, 1.5T). Default: no quota.')
         subparser.add_argument('--make-parent-dirs', dest='make_parent_dirs', action='store_true',
                                help='create the parent directories of the repository directory, if they are missing.')
+        subparser.add_argument('--key-algorithm', dest='key_algorithm', default='argon2 aes256-ctr hmac-sha256',
+                               choices=['pbkdf2-sha256 aes256-ctr hmac-sha256', 'argon2 aes256-ctr hmac-sha256'])
```

Comment:
i guess we should remove the blanks from the algorithm names so users do not need to quote the names.

also guess people will hate the long names, same issue as for -E.

Reply:
Good point! How about this?

CLI  | internal
-|-
--key-algorithm pbkdf2 | sha256
--key-algorithm argon2 | argon2 aes256-ctr hmac-sha256